test/versioned/express/segments.tap.js

```diff
@@ -940,6 +940,8 @@ function checkMetrics(t, metrics, expected, path) {
     [{ name: 'HttpDispatcher' }],
     [{ name: 'WebTransaction/Expressjs/GET/' + path }],
     [{ name: 'WebTransactionTotalTime/Expressjs/GET/' + path }],
+    [{ name: 'DurationByCaller/Unknown/Unknown/Unknown/Unknown/all' }],
```

Comment:
is this an issue that all the parent types are undefined and the other versioned tests below?

Reply:
It's in the Distributed Tracing metrics [specification](https://source.datanerd.us/agents/agent-specs/blob/d7c3a18ea32cf14bc0569d90e311e17f605b9116/distributed_tracing/Distributed-Tracing.md#metrics). If some attributes are missing for generating the metric, it can be replaced with `Unknown`. It's probably generated because of the simple server setup we're using to test with.